core/settings.py

```diff
@@ -46,6 +53,12 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
+if DEBUG:
```

Comment:
```suggestion
if "django_browser_reload" in INSTALLED_APPS:
```

Suggestion : peut-être moins utile que plus bas, mais pour être consistant on pourrait se baser sur les `INSTALLED_APPS` ici